.github/workflows/extended.yml

```diff
@@ -124,7 +124,7 @@ jobs:
           rust-version: stable
       - name: Run sqllogictest
         run: |
-          cargo test --profile release-nonlto --test sqllogictests -- --include-sqlite
+          cargo test --features backtrace --profile release-nonlto --test sqllogictests -- --include-sqlite
```

Comment:
This test isn't run on PRs (it gets run only on main). 

Did you run this test locally to verify that the tests still pass?

Reply:
I ran it locally and it passed for me.